kpi/views/environment.py

```diff
@@ -185,6 +185,7 @@ def process_other_configs(request):
                 ) from e
         else:
             data['stripe_public_key'] = None
+            data['use_team_label'] = constance.config.USE_TEAM_LABEL
```

Comment:
Let's add `data['use_team_label'] =  False` in `settings.STRIPE_ENABLED. This way we can keep the logic of the boolean in the back-end code.